anvill/python/anvill/arch.py

```diff
@@ -328,32 +329,38 @@ class AMD64Arch(Arch):
         "ST7": (("ST7", 0, 10), ("ST7", 0, 12), ("ST7", 0, 16)),
     }
 
-    def name(self):
+    def name(self) -> str:
```

Comment:
Lets update the top level to have `ArchName = str`, and start using that in place of `str` for the architecture names